### DISCUSSION TITLE
Is qdrant production ready now?

### DISCUSSION THREAD

**Discussion by dafang** [qdrant/qdrant#1351] [General]
We are leveraging qdrant, is qdrant production ready and is there any user cases?

How many Vector dimensionality is supported by qdrant?

**Comment by generall** [comment]
> We are leveraging qdrant, is qdrant production ready and is there any user cases?

Yes, there are over a hundred companies using qdrant that we know of. Of course not everybody is reporting to us.

> How many Vector dimensionality is supported by qdrant?

There is no upper limit

**Reply by dafang** [reply]
Thanks, got it

**Reply by findmyway** [reply]
By the way, is there a limitation on the maximum number of collections?

**Reply by generall** [reply]
> By the way, is there a limitation on the maximum number of collections?

There is no hard limit, but creating a collection does have some overhead, compared to the same amount of points within the single collection. 

If you need to serve different subsets per user, I would recommend to also try to do it with payload params. In this case there is a possibility to even switch off the global indexing and only build index based on payload

**Reply by findmyway** [reply]
Thanks! I'll do some benchmark test and see which approach is more suitable for my use case.
